code/modules/games/cah.dm

```diff
@@ -0,0 +1,262 @@
+	"Today, researchers found an artefact that made people ____ and ____. (Pick 2)",
+	"There was a rap battle between ____ and ____. (Pick 2)",
+	"Caution, ______ have been detected in collision course with the planet.",
+	"Today's kitchen menu includes _______.",
+	"What did space pirates want when they attacked our home?",
+	"I think our boss is insane. He just demanded ______ in his office.",
+	"Fuckin' scientists, they just turned the entire population of Scottsdan into _______ .",
+	"Fuckin' activists, they just converted Mendell City into ______ .",
+	"What's my fetish?",
+	"Next up on TCFL's Got Talent: ________ .",
+	"Next up on Biesel's World Cooking Championships: _______ .",
+	"Hello, _______ here with _______ . (Pick 2)",
+	"No one else was at _______, they wouldn't understand the ______ . (Pick 2)",
+	"Why am I shivering?",
+	"What is this world coming to? First, ________, now _______ . (Pick 2)",
+	"NanoTrasen's morale division decided to use _______ to raise employee morale.",
+	"The average junkie's drug of choice is ______ .",
+	"I saw an Eridanian dreg make out with ______ the other day.",
+	"It is common practice for _______ to ______ on Moghes. (Pick 2)",
+	"The colonies of the Coalition of Colonies are really just _____.",
+	"The Skrell are celebrating _____ today.",
+	"_____ is/are why I'm afraid of hospitals.",
+	"_____ used ____ to create their newest invention, _____!  (Draw 2; Pick 3)",
+	"Scientists are not allowed to make Gatling _____.",
+	"It's not a party until the ____ arrive.",
+	"No matter how many Tajara you have, _____ is never acceptable.",
+	"No, your robot's first law is NOT to serve _____.",
+	"The robots are not disposal bins for your _____.",
+	"You can never have too many _____ on board.",
+	"Failing to fit ______ up the _____ . (Pick 2)",
+	"_____ has no idea what S&M is and _____ shows them. (Pick 2)",
+	"What's this in my hand?",
+	"My lover left me for ______.",
+	"The _____ apocalypse comes.",
+	"Hi! I'm Malyr Krissen, and I got hit by ______.",
+	"Ah. ______ . Good times.",
+	"Well, well. If it isn't _____ .",
+	"Please welcome our newest recruit: _____ !",
+	"Didn’t you hear? Kazimir Boytsov just got sick from _____ !",
+	"\"Tau Ceti has become a beacon of liberty and hope for ______ .\"",
+	"\"This colony of the Alliance has been proliferating _____ as potential weapons of mass destruction.\"",
+	"What is the president's next speech going to be about?",
+	"It's _____ time!",
+	"That was a wild party last night. I only remember _____ .",
+	"Create a haiku. (Draw 2; Pick 3)",
+	"You can't go on like this. Go talk to the doctor about _____ .",
+	"And the award goes to...",
+	"Look! Over there! It's _____ !",
+	"What you call a passion fruit, I call a _____ .",
+	"How do I deal with my grief?",
+	"Knock knock! Who's there?",
+	"Drats! Foiled by _____ AGAIN!"
+	)
+
+// White cards.
+/obj/item/deck/cah/var/list/card_text_list = list(
+	"Those motherfucking carp",
+	"Having sex in a sewer outlet",
```

Comment:
Why I feel that this is some kind of inside joke?

Reply:
It isn't.